nomic/data_inference.py

```diff
@@ -65,8 +65,12 @@ class NomicProjectOptions(BaseModel):
     Options for Nomic 2D Dimensionality Reduction Model
 
     Args:
-        n_neighbors: The number of neighbors to use when approximating the high dimensional embedding space during reduction.
-        n_epochs: How many dataset passes to train the projection model.
+        n_neighbors: The number of neighbors to use when approximating the high dimensional embedding space during reduction. Default: `None`.
```

Comment:
Instead of saying Default: None, i would suggest saying Default: None (Auto-inferred)